### PR TITLE
Environment Init Tools.

### PR DESCRIPTION
**Pull request by parc-jason**
# Description
Experiment to use yeoman to create configmaps and secrets.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wfezkf-tools.pathfinder.gov.bc.ca/dashboard?id=document-generation-showcase-pr-22

File: tools/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+###Install
+```
+npm install -g yo@latest
+npm i
+npm link
+```
+
+###Login and Run and Clean up
+```
+oc login https://console.pathfinder.gov.bc.ca:8443 --token=RU4grkYyju_xTt78vROEGRNT6vkhps80qnx_o2RcSkY
```
Comment (jujaga) [inline tools/README.md]: Why is there a token here? Granted should be expired already...

Comment (parc-jason) [reply tools/README.md]: It's not a real token, it's there simply as an indicator that you must go get your token before using the generators.

as far as the secrets, this is the exact same create as we have been using in our projects.  i think we should move to using templates for configmaps and secrets, then we can mine the parameters and we can set a lot more useful metadata for those objects too.

**Comment by jujaga** [comment]
Closing this PR for now - when we revisit this in a future sprint we can reopen this.

**Pull request closed without merging**